{
    "name": "@duydang2311/attempt",
    "version": "0.1.10",
    "author": "duydang2311",
    "repository": {
        "type": "git",
        "url": "git+https://github.com/duydang2311/attempt.git"
    },
    "main": "dist/index.cjs",
    "module": "dist/index.js",
    "devDependencies": {
        "@types/bun": "^1.2.23",
        "agadoo": "^3.0.0",
        "bun-plugin-dts": "^0.3.0"
    },
    "peerDependencies": {
        "typescript": "^5"
    },
    "exports": {
        ".": {
            "import": "./dist/index.js",
            "types": "./dist/index.d.ts",
            "require": "./dist/index.cjs"
        }
    },
    "bugs": {
        "url": "https://github.com/duydang2311/attempt/issues"
    },
    "description": "A functional type for error handling in JS/TS",
    "files": [
        "/dist"
    ],
    "homepage": "https://github.com/duydang2311/attempt#readme",
    "keywords": [
        "error-handling",
        "try",
        "try-catch",
        "error",
        "result",
        "exception",
        "attempt",
        "functional"
    ],
    "license": "MIT",
    "scripts": {
        "build": "bun run build.ts"
    },
    "type": "module",
    "types": "dist/index.d.ts"
}
